eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/ArrayListIterate.java

```diff
@@ -109,7 +109,7 @@
                 data.setAccessible(true);
                 size.setAccessible(true);
             }
-            catch (SecurityException ignored)
+            catch (Exception exception)
```

Comment:
Same here. At first glance, it looks like we're treating the two kinds of exceptions differently because one silently ignores and the other sets to null. But the fields were initialized to null.